EMML/1001-2000/EMML1292.xml

```diff
@@ -31,20 +31,19 @@ schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://ra
          <sourceDesc>
             <msDesc xml:id="ms">
                <msIdentifier>
-                  <repository ref="INS0004HMML"/>
-                  <collection>Addis Ababa University.
-                     YaʼItyop̣yā ṭenātenā meremer taqwām, Addis Ababa</collection>
+                  <repository ref="INS0002IHA">Institute of Ethiopian Studies</repository>
```

Comment:
IES has ID INS0501IES
(all IDs that have IHA in them belong to the IslHornAfr project and should not be used by BM)